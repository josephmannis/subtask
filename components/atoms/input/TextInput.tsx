import styled from "styled-components/native";

export const Input = styled.TextInput`
    font-size: 16px;
    padding: 12px;
    background-color: #E8E8E8;
    border-radius: 7px;
`

export const InputLabel = styled.Text`
    font-size: 18px;
    font-weight: bold;
    margin-bottom: 10px;
    text-transform: uppercase;
    color: #838383;
`