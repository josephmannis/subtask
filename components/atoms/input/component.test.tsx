import React from 'react';

import renderer from 'react-test-renderer';
import { Input, InputLabel } from './TextInput';
test('Renders input', () => {
    const tree = renderer.create(<Input/>).toJSON();
    expect(tree).toMatchSnapshot()
})

test('Renders label', () => {
    const tree = renderer.create(<InputLabel>hello I'm a test!</InputLabel>).toJSON();
    expect(tree).toMatchSnapshot()
})