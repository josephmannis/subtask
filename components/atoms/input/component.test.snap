// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders input 1`] = `
<TextInput
  allowFontScaling={true}
  rejectResponderTermination={true}
  style={
    Array [
      Object {
        "backgroundColor": "#E8E8E8",
        "borderRadius": 7,
        "fontSize": 16,
        "paddingBottom": 12,
        "paddingLeft": 12,
        "paddingRight": 12,
        "paddingTop": 12,
      },
    ]
  }
  underlineColorAndroid="transparent"
/>
`;

exports[`Renders label 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#838383",
        "fontSize": 18,
        "fontWeight": "bold",
        "marginBottom": 10,
        "textTransform": "uppercase",
      },
    ]
  }
>
  hello I'm a test!
</Text>
`;
